vector-im/element-android
# Render ended polls (PSG-904)

## Type of change


## Content


We are currently not showing any separate indicator when a poll is ended. Since we may have many other messages between poll start and poll end events, users may not realize that poll is ended.

## Motivation and context
With this PR, ended polls will be rendered in timeline with winning options and total votes.



## Screenshots / GIFs
![ended_poll](https://user-images.githubusercontent.com/1254401/210782168-cff2860e-28de-4400-8570-e3f18334f979.png)





## Tests



- Create a poll
- Select an option
- Long click on poll message to show actions
- Select end poll
- See the ended poll is rendered in timeline with options, winning options, total votes and ended poll indicator text
- To test replying, select reply action of ended poll and reply with a text message

## Tested devices

- OS version(s):

## Checklist
